R/cookies.R

```diff
@@ -34,29 +34,7 @@ dfe_cookie_script <- function() {
 #' @export
 #'
 #' @examples
-#' \donttest{
```

Comment:
Was it a conscious decision to remove these comments?

Reply:
Yes, should have written in the PR description. Felt like having the whole thing there was a bit cumbersome and like it would be better suited to a vignette. Think we talked about making a vignette out of it already, but not as part of this PR.

Reply:
Cool that's fine - agree a vignette will be better for this